providers/implementations/signature/rsa.c

```diff
@@ -337,9 +337,9 @@ static int rsa_signature_init(void *vprsactx, void *vrsa, int operation)
                     return 0;
                 }
 
-                strncpy(prsactx->mdname, mdname, sizeof(prsactx->mdname));
+                strncpy(prsactx->mdname, mdname, sizeof(prsactx->mdname) - 1);
                 strncpy(prsactx->mgf1_mdname, mgf1mdname,
-                        sizeof(prsactx->mgf1_mdname));
+                        sizeof(prsactx->mgf1_mdname) - 1);
```

Comment:
This is being fixed in #11878 already.

Reply:
I suggest you read the discussion there, because this doesn't actually fix anything.

Reply:
Thanks, I'll drop this part.

Reply:
And gone...